gematria/datasets/find_accessed_addrs_exegesis.cc

```diff
@@ -32,6 +32,10 @@ using namespace llvm::exegesis;
 
 namespace gematria {
 
+// This value specifies the maximum number of times the annotator will attempt
+// to map segmentation fault addresses before giving up and throwing an error.
+constexpr unsigned kMaxAnnotationAttempts = 100;
+
```

Comment:
If it doesn't complicate things too much, this should be a parameter that can be tuned (from the command-line).

Reply:
Done. It's now a parameter in the BHive to Exegesis conversion script.